### ISSUE TITLE
check-source BOLT mismatch

### ISSUE THREAD

**Issue by jsarenik** [ElementsProject/lightning#1575]
### Issue and Steps to Reproduce

I tried to clone also https://github.com/lightningnetwork/lightning-rfc and see what happens when I run `make check-source`. Here it is:

```
if [ x"`LC_ALL=C ls common/*.h | grep -v ^common/gen_`" != x"`echo common/base32.h common/bech32.h common/bech32_util.h common/bip32.h common/bolt11.h common/channel_config.h common/close_tx.h common/configdir.h common/crypto_state.h common/crypto_sync.h common/cryptomsg.h common/daemon.h common/daemon_conn.h common/derive_basepoints.h common/dev_disconnect.h common/features.h common/funding_tx.h common/hash_u5.h common/htlc_state.h common/htlc_tx.h common/htlc_wire.h common/initial_channel.h common/initial_commit_tx.h common/json.h common/json_escaped.h common/key_derive.h common/keyset.h common/memleak.h common/msg_queue.h common/peer_billboard.h common/peer_failed.h common/permute_tx.h common/ping.h common/pseudorand.h common/read_peer_msg.h common/socket_close.h common/sphinx.h common/status.h common/status_wire.h common/subdaemon.h common/timeout.h common/type_to_string.h common/utils.h common/utxo.h common/version.h common/wallet_tx.h common/wireaddr.h common/wire_error.h common/withdraw_tx.h common/overflows.h common/htlc.h common/status_levels.h | tr ' ' '\n' | LC_ALL=C sort`" ]; then echo COMMON_HEADERS_NOGEN incorrect; exit 1; fi
bitcoin/script.c:488:mismatch:/* BOLT #3:
Closest match: #### Offered HTLC Outputs This output sends funds to a...[ HTLC-timeout transa]
Makefile:236: recipe for target 'bolt-check/bitcoin/script.c' failed
make: *** [bolt-check/bitcoin/script.c] Error 1
```

See https://github.com/jsarenik/lightning-auto-test/tree/master/run.sh for a full example how I run the tests.

**Comment by cdecker** [comment]
Yep, that is a known issue right now. Since the BOLTs were being heavily copy-edited we disabled the check for BOLT quotes, and haven't re-enabled them yet. Once the edits settle down we can go through the source and adjust the quotes to match the BOLT text.

**Comment by rustyrussell** [comment]
Finally fixed by #1640

**Comment by jsarenik** [comment]
Works well. Thanks. https://gist.github.com/jsarenik/3fc5808a71d82012bb7314116a587c7e
